### DISCUSSION TITLE
Transient problem custom stop condition

### DISCUSSION THREAD

**Discussion by ermore** [idaholab/moose#25094] [Q&A General]
Let me describe my problem first, my calculation is a transient problem, and the calculation time is generally 0 to t1. I will use postprocessors to monitor some variables during the calculation process, such as var. I want the calculation to stop when var reaches a certain value. What should I do?

**Comment by GiudGiud** [comment]
Hello

Use the Terminator user object to have a termination criterion based on postprocessors
https://mooseframework.inl.gov/source/userobjects/Terminator.html

Guillaume
